### ISSUE TITLE
DualVideoDemo doesn't autoplay anymore

### ISSUE THREAD

**Issue by happylinks** [remotion-dev/remotion#3346]
# Bug Report 🐛

I noticed the videos on this demo page don't autoplay anymore for me: https://www.remotion.dev/docs/animating-properties
Looking at the [code](https://github.com/remotion-dev/remotion/blob/4dcc1ca4e40f94ba6a8a8b66a6f11e3fc7fcf19d/packages/docs/components/DocsDark.tsx#L64) I don't see any recent changes, so maybe it's because of a chrome change?

Adding autoPlay=true would probably fix it, but I thought I'd check first if that's the behavior you wanted before making a PR.

**Comment by Ali7040** [comment]
@happylinks But in my case, it's working fine. Could you please provide more details?

**Comment by happylinks** [comment]
<img width="978" alt="image" src="https://github.com/remotion-dev/remotion/assets/184774/02aa7680-4040-46b7-8dc5-99218e26a60a">

Chrome is up to date
Version 120.0.6099.216 (Official Build) (arm64)

Just tested Firefox and it has the same behavior for me, only Safari works properly.

I think maybe it's because you won't have enough "user interaction" yet on the docs site if you go to the page directly. If I navigate around the docs and go back to the animating-properties page, then they play properly.

Also, they show up in light mode on refresh (server-side), and dark mode when I navigate from the menu (client-side).

Not sure if that helps, let me know if I can test anything else.

**Comment by JonnyBurger** [comment]
Indeed, I still had Chrome 119, once I updated, it did not work anymore

I'll try #3356
The reason we don't use autoPlay is because we want to start both videos at the same time
